### PR TITLE
Fix message logic in Docker installation script: handle existing keyr…

### PR DESCRIPTION
**Pull request by zaylon10**
…ings correctly.

### CODE REVIEW

File: containerization-installs/docker-install.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ fi
 # Creating a keyrings directory 
 if (test -d /etc/bin/keyrings)
 then
-  echo "creating keyrings"
+  echo "keyrings directory already exist"
```
Comment (ChrisForti) [inline containerization-installs/docker-install.sh:31]: Looks good to me.  For the sake of being 
pedantic, might as well change the comment in the else statement as well....

**Review by ChrisForti — CHANGES_REQUESTED** [review]

File: containerization-installs/docker-install.sh [hunk 2]
```diff
@@ -65,4 +65,4 @@ then
 else
   echo "Installing docker"
   sudo apt install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
```
Comment (stephenlauck) [inline containerization-installs/docker-install.sh:67]: I would break each package install into an apt-cache show check and then apt install.

**Review by stephenlauck — DISMISSED** [review]

File: containerization-installs/docker-install.sh [hunk 3]
```diff
@@ -28,7 +28,7 @@ fi
 # Creating a keyrings directory 
 if (test -d /etc/bin/keyrings)
```
Comment (thatoldplatitude) [inline containerization-installs/docker-install.sh:29]: Paths in the if(test...) don't match what gets created in the else. Ik this wasn't a change in this pr, but was this done on purpose?  
Same with ln40

**Comment by thatoldplatitude** [comment]
ln53-ln59 are dupes of ln2-ln8.
Maybe make a seperate ticket for that, as well as my other comment.
Unless I'm missing something

**Comment by zaylon10** [comment]
No, I did not change any of his code ,because he made a great script from what I reviewed. I just helped with the wording change fixing any inconsistency.

File: containerization-installs/docker-install.sh (hunk 3 above)
Comment (ChrisForti) [reply containerization-installs/docker-install.sh:29]: A lot of this was just slapped together months ago. So it's is very likely not right, thank you Greg for the input👍

File: containerization-installs/docker-install.sh (hunk 2 above)
Comment (ChrisForti) [reply containerization-installs/docker-install.sh:67]: Yes that sounds like a plan.

**Comment by ChrisForti** [comment]
Fixed your concerns @thatoldplatitude , @stephenlauck . The duplicate update commands are directly from docker.docs. 
1. to get any new pkgs you may not have.
2. Then i believe the second refreshes any new material(not entirely sure), these commands came from docker in this specific order.

**Comment by thatoldplatitude** [comment]
>     1. to get any new pkgs you may not have.
> 
>     2. Then i believe the second refreshes any new material(not entirely sure), these commands came from docker in this specific order.
> 

The first one is to make sure apt has the most current versions of packages available. e.g. if you haven't updated in a while and the last version of `curl` apt knew of was `1.0` but the most recent version is `2.0`, you'd be installing an outdated pkg.

The second update is because you've just added a new repo into apt's repo list. So you need to run `apt update` so apt will know of the new repo you just added.


So you should have 2 `apt update`s, I was just saying how they were wrapped in `if`s was duped.

File: containerization-installs/docker-install.sh [hunk 4]
```diff
@@ -28,41 +28,91 @@ fi
...
+then
+  echo "The Docker GPG key already exists"
+else
+  echo "The Docker GPG key does not exist. Downloading it"
+  sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+fi
  
 # setting permissions 
-if (test -d /etc/bin/keyrings/docker.asc)
+if (-r /etc/apt/keyrings/docker.asc)
```
Comment (thatoldplatitude) [inline containerization-installs/docker-install.sh:56]: Idt you meant to delete the `(test ...)` part of this line

Comment (ChrisForti) [reply containerization-installs/docker-install.sh:56]: You are right greg thanks.

**Review by ChrisForti — APPROVED** [review]
I sign off👍🏼 as an minimum viable product.

**Comment by ChrisForti** [comment]
Probably should not have done this, I did not finish my commit concerning the typo Greg caught. I will stash and bundle that into the next branch.

**Pull request merged**